### ISSUE TITLE
Gaps in some graphs and graph overextending arrows

### ISSUE THREAD

**Issue by drgrice1** [jsxgraph/jsxgraph#344]
I have noticed gaps in some graphs (particularly cubics or cubed roots).  This only occurs if firstArrow is true, and is made quite noticeable with a larger strokeWidth.

Here is an example:
```javascript
var board = JXG.JSXGraph.initBoard("box", { boundingbox: [-8, 4, 8, -4] });
board.create('curve', [
	function(t) { return t; },
	function(x) { return Math.abs(x) / x * Math.pow(Math.abs(x), 1 / 3); },
	-7, 7
], { strokeWidth: 5, firstArrow: true, lastArrow: true });
```

The lastArrow option does not affect the gap at the origin, but I included it because in this example you also see the curve extending past the end of the arrow.

This happens in version 1.1.0 and 1.2.1 (as well as for a build of the 1.2.2 branch) of jsxgraph, but does not happen in version 1.0.0.  Notice the plotVersion: 4 makes the gap even larger.

**Comment by drgrice1** [comment]
Hmm.  Maybe this is an error in my function definition.  The function is undefined at x = 0.  I guess version 1.0.0 graphed over that undefined point, but the newer versions of jsxgraph do not.

**Comment by drgrice1** [comment]
I am going to leave this issue open, because the gap still happens for plotVersion: 4, even if you instead define the function by
```javascript
function(x) { return Math.sign(x) * Math.pow(Math.abs(x), 1 / 3); }
```
which, of course, is defined for all real numbers.

**Comment by alfredwassermann** [comment]
I agree, this plot looks terrible!

**Comment by alfredwassermann** [comment]
Fixed with 9d0e2d10ed4abf0c7fa4c0126b0ee5b81684f2f5
